mne/simulation/evoked.py

```diff
@@ -66,14 +74,17 @@ def simulate_evoked(fwd, stc, info, cov, snr=3., tmin=None, tmax=None,
     -----
     .. versionadded:: 0.10.0
     """
+    if snr is not None:
+        warn('snr is deprecated and will be removed in 0.16. Set it to '
+             'None to remove this warning. Set nave parameter instead.',
+             DeprecationWarning)
```

Comment:
This isn't a proper deprecation. If `snr` is passed, it should still be used equivalently as before. Right now it's just ignored.

Once you have the lines I mention above about converting snr<->nave, then you can simply add the line here as `nave = ...` and everything should work properly.